Makefile

```diff
@@ -384,11 +384,14 @@ agbot-package: agbot-image
 	fi
 
 anax-k8s-image: anax-k8s-clean
-	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)
-	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)
-	cp -f $(LICENSE_FILE) $(ANAX_K8S_CONTAINER_DIR)
-	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)"
-	cd $(ANAX_K8S_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(ANAX_K8S_IMAGE_LABELS) -t $(ANAX_K8S_IMAGE_STG) -f Dockerfile.ubi . && \
+	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR);
+	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR);
+	cp -f $(LICENSE_FILE) $(ANAX_K8S_CONTAINER_DIR);
+	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)";
```

Comment:
Why did you add `;` to these lines? That seems unnecessary.

Reply:
Sorry, somehow accidentally got there, by autoformat

Reply:
fixed